magefiles/lint.go

```diff
@@ -88,8 +94,15 @@ func (Lint) Analyzers() error {
 	)
 }
 
-// Run vulncheck
+// Vulncheck Run vulncheck
 func (Lint) Vulncheck() error {
 	fmt.Println("running vulncheck")
 	return sh.RunV("go", "run", "golang.org/x/vuln/cmd/govulncheck", "./...")
 }
+
+// Trivy Run Trivy
+func (l Lint) Trivy() error {
+	mg.Deps(Build{}.Testimage)
+	fmt.Println("running Trivy container scan")
+	return sh.RunV("docker", "run", "-v", "/var/run/docker.sock:/var/run/docker.sock", "aquasec/trivy:latest", "image", "--format", "table", "--exit-code", "1", "--ignore-unfixed", "--vuln-type", "os,library", "--no-progress", "--severity", "CRITICAL,HIGH,MEDIUM", "authzed/spicedb:ci")
```

Comment:
Since trivy is a go project, I think we could manage it as a `tool` and just run with `go run github.com/aquasecurity/trivy/cmd/trivy/main.go`?